README.md

```diff
@@ -132,16 +132,70 @@ SaaSKit comes with `primary` and `secondary` colors predefined within
 
 ## Deploying to Production
 
-### Authentication
+This section assumes that a
+[local development environment](#getting-started-locally) has been set up.
 
-TODO
+### Database (Supabase)
 
-### Customer Portal Branding (Stripe)
+We'll use the Supabase CLI to migrate your local database to Supabase's remote
+production servers.
 
-[Set up your branding on Stripe](https://dashboard.stripe.com/settings/branding),
-as the user will be taken to Stripe's checkout page when they upgrade.
+1. Login with the Supabase CLI
+
+```sh
+supabase login
+```
+
+2. Run the Supabase link command to setup a local-remote project link
+
+- Get project-ref from the last part of the Supabase SaaSKit project URL:
+  https://app.supabase.com/project/{{ saaskit project-ref }}
+
+- Run `supabase link`:
+
+```sh
+supabase link --project-ref {{ supabase project ref }}
+```
+
+3. Run `supabase db push` to push the migration SQL (found in
+   `supabase/migrations`) to your remote Supabase account.
+
+```sh
+supabase db push -p {{ supabase project pwd }}
```

Comment:
```suggestion
supabase db push
```
Again, I believe `supabase login` automatically processes the password, so there's no need to define it explicitly.